Config/bootstrap.php

```diff
@@ -0,0 +1,4 @@
+<?php
+if(!Configure::check('AuditLog.data_source')) {
```

Comment:
While I personally like configuring my plugins through Configure, this introduces a second way of configuration besides the established one, as documented here:

https://github.com/robwilkerson/CakePHP-Audit-Log-Plugin#configuration